src/lib/flexbox/api/base.ts

```diff
@@ -127,7 +135,7 @@ export abstract class BaseFxDirective implements OnDestroy, OnChanges {
     let value = this._lookupStyle(element, 'display');
 
     // Note: 'inline' is the default of all elements, unless UA stylesheet overrides
-    return value ? value.trim() : 'inline';
+    return value ? value.trim() : ((element.nodeType === 1) ? 'block' : 'inline');
```

Comment:
That won't be true necessarily for all cases. One potential solution can be to have a list of the default block-level elements and verify it against the `element.nodeName`, assuming that everything else is `inline`.